styles/style.css

```diff
@@ -120,8 +161,16 @@ p {
 /* hero */
 
 .hero {
-  background-image: url("header-bike.jpg");
+  background-image: url("/images/header-bike.jpg");
   background-color: var(--grey-light);
+  height: 58%;
+  background-size: auto;
```

Comment:
I meant here by my comment earlier how about if we use this:
.hero{
background-size: contain;
}
when the "background-size is set to auto, the image still is cropped

Reply:
after doing some adjustments I found out we could do like this 
background-size: 100% 100%;
this property would stretch the image to fill the entire hero container.
the auto property the image would be cropped to fit the container by using the container width and height and ignoring the images width and height. If I'm not wrong :wrench:

Reply:
Thank you for noticing this Abdi! I'll fix this now